docs/AboutUs.md

```diff
@@ -57,3 +57,13 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Developer
 * Responsibilities: Integration, Scheduling and Tracking
+
+### Luo Siyi
+
+<img src="images/luo.png" width="200px">
```

Comment:
According to the CS2103T website, I think you should rename your image file to be <github-username>.png?